www/modules/centreon-open-tickets/views/rules/ajax/actions/scheduleCheck.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+/*
+ * Copyright 2016-2019 Centreon (http://www.centreon.com/)
+ *
+ * Centreon is a full-fledged industry-strength solution that meets
+ * the needs in IT infrastructure and application monitoring for
+ * service performance.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,*
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+$resultat = array(
+    "code" => 0,
+    "msg" => 'ok',
+);
+
+// We get Host or Service
+$selected_values = explode(',', $get_information['form']['selection']);
+$forced = $get_information['form']['forced'];
+$isService = $get_information['form']['isService'];
+$db_storage = new centreonDBManager('centstorage');
+
+$problems = array();
+
+# check services and hosts
+$selected_str = '';
+$selected_str_append = '';
+$hosts_selected_str = '';
+$hosts_selected_str_append = '';
+$hosts_done = array();
+$services_done = array();
```

Comment:
```suggestion
$hostsDone = [];
$servicesDone = [];
```
Can you use camelCase as naming convention and use brace instead of `array()` for all other variables? 🙏